src/models/converters/transactions/mod.rs

```diff
@@ -179,6 +179,7 @@ impl MultisigTransaction {
             to: self.to.to_owned(),
             data_size: data_size(&self.data),
             value: self.value.as_ref().unwrap().into(),
+            method_name: self.data_decoded.as_ref().map(|it| it.method.clone()),
```

Comment:
Maybe we should at some point define how we convert Strings into owned strings. in some places we use `.to_owned()` in some `.clone()` ... not really important and not sure about the difference. (`to_owned()` works with `&str` and `String` :D)

Reply:
yeah, I would read on the implications of either option, I just changed it `to_owned` to keep it consistent for now :)